.changeset/loud-snails-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-vault-backend': patch
+---
+
+Added (optional) public link option to entity vault card in case base link is internal
```

Comment:
I originally had:
`Added (optional) publicUrl to editUrl and showUrl in case baseUrl is an internal url`

But vale did not like it, as I was using many unknown words. Not sure what the practice is here.
I assumed you don't want to add all the config names in `.github/vale/Vocab/Backstage/accept.txt`